docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,156 @@
+.. _supported_tasks_and_formats:
+
+===========================
+Supported Tasks and Formats
+===========================
+
+In order to analyze the images, labels and predictions provided in the
+:doc:`VisionData object </user-guide/vision/vision_data_class>`,
+they must be provided the accepted deepchecks format. In this section we will describe the supported formats
+for each supported task type.
+
+The VisionData object automatically validates your data formats when initialized, using the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
+correct format via the :func:`head <deepchecks.vision.vision_data.VisionData.head>` functionality.
```

Comment:
also this link is broken...
I think after API ref fix we have to manually check these as we don't receive warnings :-/